hw_diag/utilities/hardware.py

```diff
@@ -1,11 +1,70 @@
 import logging
-import os
 from time import sleep
-
+import dbus
 from hm_pyhelper.miner_param import get_public_keys_rust
 from hm_pyhelper.hardware_definitions import variant_definitions, is_rockpi
 from hw_diag.utilities.shell import config_search_param
 
+log = logging.getLogger()
```

Comment:
Can you use get_logger from hm_pyhelper?

Reply:
Ok, make sense.